lang/syn/src/codegen/accounts/constraints.rs

```diff
@@ -37,14 +37,15 @@ pub fn generate_composite(f: &CompositeField) -> proc_macro2::TokenStream {
         #(#checks)*
     }
 }
-
+//TODO[paulx]: where to add Dup
```

Comment:
This is fine as is. Constraints like init need to run before other constraints since they initialize the accounts and require special handling.